test/loaders.js

```diff
@@ -327,3 +327,63 @@ test('support context in loader', async (t) => {
     }
   })
 })
+test('rersolver unknown type', async t => {
+  const app = Fastify()
+
+  const resolvers = {
+    test: 2
+  }
+
+  app.register(GQL, {
+    resolvers
+  })
+
+  const typeError = new Error('Cannot find type test')
+
+  try {
+    // needed so that graphql is defined
+    await app.ready()
+    app.graphql('query { test }')
+  } catch (error) {
+    t.deepEqual(error, typeError)
```

Comment:
Can you avoid creating a new error and just check the message on the generate one?